test/functional/test_cmdline.jl

```diff
@@ -0,0 +1,15 @@
+using Test, MPI
+
+@testset "cmdline" begin
+    mpiexec() do runcmd
+        juliacmd = `julia --project=.`
+
+        @test run(
+            Cmd(`$runcmd -n 4 $juliacmd riopa.jl --hello`),
```

Comment:
Thanks @PhilipFackler . I think these kind of tests are valuable when we have actual config.yaml tied to HPC runs.